src/Microsoft.SqlTools.ServiceLayer/SchemaCompare/SchemaCompareOperation.cs

```diff
@@ -113,6 +113,9 @@ public void Execute(TaskExecutionMode mode)
                 this.Differences = new List<DiffEntry>();
                 if (this.ComparisonResult.Differences != null)
                 {
+                    // filter out not included and not excludeable differences
+                    (this.ComparisonResult.Differences as List<SchemaDifference>).RemoveAll(d => !d.Included && !d.IsExcludable);
+
```

Comment:
Would this also fix the issues where the script generation has the excludable stuff in?

Reply:
I'm not sure. It might if those actually aren't excludable. We can ask the customer to try in insiders after this is checked in or ask them for a repro so we can test it out.

Reply:
Sounds good.